index.js

```diff
@@ -1313,7 +1338,12 @@ module.exports = function (log, indexesPath) {
     else return bitset.size() - seq
   }
 
-  function paginate(operation, seq, limit, descending, onlyOffset, cb) {
+  function paginate(operation, seq, limit, descending, onlyOffset, sortBy, cb) {
+    if (!cb) {
```

Comment:
I ended up defaulting this so we don't have to break the API of `paginate` and `all`. Can change if needed.

Reply:
I would tend towards a breaking change, and bumping ssb-db2, it doesn't take that much work. But maintaining this legacy mode of either 6 or 7 arguments seems like something we could burn ourselves with in the future.

Reply:
@arj03 How about this?

Reply:
ohh noo. I didn't see this. I'll do a new PR with a breaking change and publish that as 5.0?